src/app/search/query.js

```diff
@@ -21,18 +21,17 @@ const addFieldQuery = (queryString, field = 'keyword') => {
     throw new Error('A valid query string must be passed');
   }
   let fieldQuery = [];
-  // Strip punctuation and process spaces as plus signs for final split.
-  const queryArr = queryString.replace(/[.\/#!$%\^&;{}=\-_`~()]/g, '').trim().replace(/\s+/g, '+').split('+');
   /**
-   * Reformat the string.
-   * For multiple strings in a query, add the plus operator to AND terms together;
-   * otherwise, return a single string.
+   * Strip punctuation and process spaces as plus signs for final split.
+   * ES characters to escape before sending: + - = && || > < ! ( ) { } [ ] ^ " ~ * ? : \ /
+   * For multiple strings in a query, join with a space.
    */
-  const esQuery = (queryArr.length > 1) ? queryArr.join(' +') : queryArr.join('');
+  const queryArr = queryString.replace(/[=(&&)(||)><!(){}\[\]^"~\*\?:\/-]/g, '\\$&').trim().replace(/\s+/g, '+').split('+');
```

Comment:
The conflict seems to be coming from here, specifically `\\$&`, where the double backslash is causing the ElasticSearch parser to excessively escape, resulting in query strings such as `\\\\\\\"World War\\\\\\\" OR \\\\\\\"Divine Comedy\\\\\\\"`. Essentially the API is reading the string as an escaped backslash followed by a double quote, will look at if I can force it to parse that properly.